build-arch-gce

```diff
@@ -163,18 +153,18 @@ arch-chroot -- "$mount_dir" /bin/bash -s <<-'EOS'
 	rm /boot/initramfs-linux-fallback.img
 	mkinitcpio --nocolor --preset linux
 
-	echo '-- Configuring grub.'
-	grub-install --target=x86_64-efi --efi-directory=/efi --no-nvram --removable
-	cat <<-'EOF' > /etc/default/grub
-		# GRUB boot loader configuration
-		GRUB_CMDLINE_LINUX="console=ttyS0,38400n8 net.ifnames=0 scsi_mod.use_blk_mq=Y"
-		GRUB_PRELOAD_MODULES="part_gpt"
-		GRUB_TIMEOUT=0
-		GRUB_DISABLE_RECOVERY=true
-	EOF
-	grub-mkconfig -o /boot/grub/grub.cfg
+	echo '-- Running boot loader.'
+	bootctl install
```

Comment:
I would also pass --no-variables as I'm afraid it might change the local computer config otherwise. Also --quiet.

Reply:
Ack.